x-pack/platform/plugins/shared/onechat/server/services/data/data_service.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { Logger } from '@kbn/core/server';
+import type { KibanaRequest } from '@kbn/core-http-server';
+import type { Runner } from '@kbn/onechat-server';
+import {
+  createDataTypeRegistry,
+  type DataTypeRegistry,
+  type DataTypeDefinition,
+} from './data_type_registry';
+
+export interface DataServiceSetupDeps {
+  logger: Logger;
+}
+
+export interface DataServiceStartDeps {
+  getRunner: () => Runner;
+}
```

Comment:
I don't think you need (or will ever need) to access the `Runner` from the data service? That dep can probably be removed.

Reply:
Fair enough, I'll remove it. The `runner` abstraction is one I haven't gotten my head around, I thought it was sth every service would need.

Reply:
Removed in [71a2b35](https://github.com/elastic/kibana/pull/231653/commits/71a2b35aa6a49e881ec87afe7695c8ec1b1a0f29)